github/pulls_test.go

```diff
@@ -232,31 +233,63 @@ func TestPullRequestsService_Create_invalidOwner(t *testing.T) {
 }
 
 func TestPullRequestsService_Edit(t *testing.T) {
-	setup()
-	defer teardown()
+	tests := []struct {
+		input        *PullRequest
+		sendResponse string
+
+		wantUpdate   *pullRequestUpdate
```

Comment:
This is optional, but I think it'd be better if you made the type of `wantUpdate` a `string` and have it contain the expected JSON.

There's no need to decode the update JSON into our own struct that we used for encoding, and compare those.

What do you think?